### PR TITLE
Configure rubocop and add to default rake task. Ignore files we don't…

### PR DESCRIPTION
**Pull request by patbenatar**
… control.

@jdwolk @cltatman

### CODE REVIEW

**Comment by patbenatar** [comment]
For #4

File: lib/suspenders/app_builder.rb [hunk 1]
```diff
@@ -422,7 +422,6 @@ def setup_segment
 
     def setup_bundler_audit
       copy_file "bundler_audit.rake", "lib/tasks/bundler_audit.rake"
-      append_file "Rakefile", %{\ntask default: "bundler:audit"\n}
```
Comment (patbenatar) [inline lib/suspenders/app_builder.rb:425]: This appeared to be a bug in suspenders.. it was setting the default rake task to `spec`, then setting it again to `bundler:audit` :-/

File: templates/.rubocop.yml [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+DoubleNegation:
+  Enabled: false
+
+AllCops:
+  Exclude:
+    - "bin/**/*"
+    - "db/**/*"
+    - "config/**/*"
+    - "Rakefile"
+    - "spec/i18n_spec.rb"
```
Comment (patbenatar) [inline templates/.rubocop.yml:25]: - `i18n_spec` is copied from a dependency gem of suspenders, so we don't control it
- `Rakefile` by default in rails has a line that's too long
- `config/**/*` has all kinds of violations by default in Rails

Comment (jdwolk) [inline templates/.rubocop.yml:25]: :clap: so tired of fixing default comments due to line length violations...better to just ignore :laughing:

**Comment by jdwolk** [comment]
:thumbsup: looks good to me. @cltatman :question:

File: templates/.rubocop.yml (hunk 2 above)
Comment (patbenatar) [inline templates/.rubocop.yml:25]: Yeah. `config` == wild west

Comment (taboularasa) [inline templates/.rubocop.yml:25]: I just do:

```
AllCops:
  Exclude:
    - "*/**/*"
```

Works great for productivity.

Comment (patbenatar) [inline templates/.rubocop.yml:25]: Good call @taboularasa

**Pull request merged**